ccdproc/__init__.py

```diff
@@ -30,3 +39,7 @@ class Conf(_config.ConfigNamespace):
             'when calling processing operations.'
             )
     conf = Conf()
+
+
+# NOTE: the test runner is defined in _astropy_init
+__all__ = ['test', 'get_distribution', 'DistributionNotFound']
```

Comment:
Why do we need to add an `__all__` in here? Seems like this may break some code if someone does `from ccdproc import *`?

Reply:
``get_distribution`` and ``DistributionNotFound`` should definitely not be in ``__all__`` since they are not in the public API for ccdproc but just conveniences for getting the version number. Also I agree that currently this will prevent users from seeing functions like ``create_deviation`` and so on.